rfml.go

```diff
@@ -476,6 +464,120 @@ func uploadRFMLFilesInDirectory(rfmlDirectory string) error {
 	return nil
 }
 
+func downloadRFML(c cliContext) error {
+	testDirectory := c.String("test-folder")
+	absTestDirectory, err := prepareTestDirectory(testDirectory)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	var mappings rainforest.TestIDMappings
+	mappings, err = api.GetRFMLIDs()
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	var testIDs []int
+	if len(c.Args()) > 0 {
+		var testID int
+		for _, arg := range c.Args() {
+			testID, err = strconv.Atoi(arg)
```

Comment:
Probably better to use `strconv.ParseUint` here: https://golang.org/pkg/strconv/#example_ParseUint

As we cant have negative testIds